WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -322,6 +328,17 @@ private void refreshPeopleListFragment() {
     private boolean navigateBackToPeopleListFragment() {
         if (getFragmentManager().getBackStackEntryCount() > 0) {
             getFragmentManager().popBackStack();
+
+            // We need to reset the toolbar elevation if the user is navigation back from PersonDetailFragment
+            FragmentManager fragmentManager = getFragmentManager();
+            PersonDetailFragment personDetailFragment = (PersonDetailFragment) fragmentManager
```

Comment:
There are a number of places in this activity where `findFragmentByTag` is used to locate the list or detail fragment. I think it would be cleaner to have methods for each instead, like this:
```
private PeopleListFragment getListFragment() {
    return (PeopleListFragment) getFragmentManager().findFragmentByTag(KEY_PEOPLE_LIST_FRAGMENT);
}

private PersonDetailFragment getDetailFragment() {
    return (PersonDetailFragment) getFragmentManager().findFragmentByTag(KEY_PERSON_DETAIL_FRAGMENT);
}
```